content/docs/instrumenting/writing_exporters.md

```diff
@@ -399,17 +399,17 @@ docs.
 
 ### Pushes
 
-Some applications and monitoring systems only push metrics e.g. statsd,
-graphite and collectd.
+Some applications and monitoring systems only push metrics e.g. `statsd`,
+`graphite` and `collectd`.
 
 There’s two considerations here.
 
-Firstly, when do you expire metrics? Collected and things talking to Graphite
+Firstly, when do you expire metrics? `collectd` and things talking to `graphite`
```

Comment:
Graphite is a proper noun, it gets a capital letter. None of these are code, so they don't get backticks. StatsD is the preferred capitalisation of statsd.